Gems/AWSGameLift/cdk/aws_gamelift/flex_match/flex_match_configurations.py

```diff
@@ -0,0 +1,34 @@
+"""

```

Comment:
Minor: btw, can we put this configuration into fleet_configuration.py, it maybe better to keep all configs inside single file?

Reply:
Talked offline. Keep it as a separate config file since the FlexMatch feature is optional.